frontend/scripts/metrica.js

```diff
@@ -0,0 +1,231 @@
+(() => {
+    let state = {
+        host: 'http://stage.metrica.fun/api/v1',
+        routes: {
+            create_visitor: '/visitors',
+            create_visit: '/visits'
+        }
+    };
+
+    const getTrackingId = () => {
+        let scripts = document.getElementsByTagName('script');
+        let index = scripts.length - 1;
+        let myScript = scripts[index];
```

Comment:
are you sure that script will be the last on a page? Probably it is better to use querySelector() or find by a value of source, something like this "[src=https://metrica.fun/script.js?]". Or, as it also will be on the stage we can put to instruction set tracking number globally, during the initialization.

window._metricaTracking = {
      trackingNumber: <>,
      startTime: Date.now()
};